isis/src/control/objs/ControlPoint/ControlPoint.cpp

```diff
@@ -959,7 +959,7 @@ namespace Isis {
 
     // if point is Fixed or Constrained in any number of coordinates, initialize adjusted surface
     // point to a priori coordinates (set in e.g. qnet or cneteditor) and exit
-    if( IsFixed() || IsConstrained()) {
+    if( IsFixed() || IsConstrained() || id.contains("Lidar")) {
```

Comment:
I mentioned this in issue #4703, but I think that this should be refomred to have the Lidar points come in as constrained points rather than having this pointid check, as pointid is specified by the user on read in of the Lidar data.

Reply:
For right now, I want to leave this as is to ensure we're reproducing but I'm going to make a new issue to update this after we validate.